packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -396,15 +396,21 @@ module.exports = class ImportEntries {
                           }
                         })
                         .catch((error) => {
-                          if (error.hasOwnProperty('error_code') && error.error_code === 119) {
+                          if (error.hasOwnProperty('errorCode') && error.errorCode === 119) {
                             if (error.errors.title) {
                               log(
                                 this.config,
                                 'Entry ' + eUid + ' already exist, skip to avoid creating a duplicate entry',
                                 'error',
                               );
                             } else {
-                              log(this.config, `Failed to create an entry ${eUid} ${formatError(error)}`, 'error');
+                              log(
+                                this.config,
+                                `Failed to create an entry ${eUid} ${formatError(error)} Title of the failed entry: ${
+                                  entries[eUid].title
```

Comment:
@cs-raj Please use the quotes for the entity name for better readability.

Reply:
Done